### PR TITLE
Glossary

### PR DESCRIPTION
**Pull request by MohammadAlhalaq**
Glossary page
#46

### CODE REVIEW

File: client/src/components/pages/Glossary/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './style.css';
+import table from '../../../assets/table.svg';
+
+function glossary() {
```
Comment (AbdallahAmmar96) [inline client/src/components/pages/Glossary/index.js]: Use ES6

File: client/src/components/pages/Glossary/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  return (
+    <div>
+      <h1>Glossary</h1>
```
Comment (AbdallahAmmar96) [inline client/src/components/pages/Glossary/index.js]: Where is the calss name ?

File: client/src/components/pages/Glossary/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      <p>You can find Glossary of Therap here to know .</p>
+      <img src={table} alt="table" />
```
Comment (AbdallahAmmar96) [inline client/src/components/pages/Glossary/index.js]: Put a better alt

File: client/src/components/pages/Glossary/index.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './style.css';
+import table from '../../../assets/table.svg';
+
+function glossary() {
+  return (
+    <div>
```
Comment (AbdallahAmmar96) [inline client/src/components/pages/Glossary/index.js]: don't use div

File: client/src/components/pages/Glossary/style.css [hunk 5]
```diff
@@ -0,0 +1,17 @@
+.glossary_title{
```
Comment (AbdallahAmmar96) [inline client/src/components/pages/Glossary/style.css]: are you sure the style is working?
I don't think so

File: client/src/components/pages/Glossary/index.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
Comment (AbdallahAmmar96) [inline client/src/components/pages/Glossary/index.js]: add a new line

File: client/src/components/pages/Glossary/style.css [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+  padding: 0.5rem 0.5rem 0 0.5rem;
+}
+
+.glossary_description {
+  font-family: sans-serif;
+  padding: 0 1.5rem 0.5rem 0.5rem;
+}
+
+
+.glossary_table{
```
Comment (hashemt) [inline client/src/components/pages/Glossary/style.css]:
```suggestion
.glossary_table {
```

File: client/src/components/pages/Glossary/style.css (hunk 5 above)
Comment (hashemt) [inline client/src/components/pages/Glossary/style.css]:
```suggestion
.glossary_title {
```

File: client/src/components/pages/Glossary/style.css [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+  font-family: sans-serif;
+  padding: 0.5rem 0.5rem 0 0.5rem;
+}
+
+.glossary_description {
+  font-family: sans-serif;
+  padding: 0 1.5rem 0.5rem 0.5rem;
+}
+
+
```
Comment (hashemt) [inline client/src/components/pages/Glossary/style.css]: delete this emtpy line.

**Pull request merged**